low-level/files/opt/seagate/sspl/prerequisites/sspl_dev_deploy.sh

```diff
@@ -0,0 +1,484 @@
+    yum remove -y cortx-sspl.noarch
+    exit ;
+}
+
+
+# Setup common, 3rd_party and build specific repos
+[ "$setup_repo" == true ] && {
+    curl ${CORTX_MONITOR_BASE_URL}/low-level/files/opt/seagate/sspl/prerequisites/setup_yum_repos.sh -o setup_yum_repos.sh
+    chmod a+x setup_yum_repos.sh
+
+    if [ "$disable_sub_mgr" == "true" ] && [ -n "$TARGET_BUILD" ]; then
+        ./setup_yum_repos.sh -d -t $TARGET_BUILD 2>&1 | tee -a "${LOG_FILE}"
+    elif [ "$disable_sub_mgr" == "true" ]; then
+        ./setup_yum_repos.sh -d 2>&1 | tee -a "${LOG_FILE}"
+    elif [ -n "$TARGET_BUILD" ]; then
+        ./setup_yum_repos.sh -t $TARGET_BUILD 2>&1 | tee -a "${LOG_FILE}"
+    else
+        ./setup_yum_repos.sh 2>&1 | tee -a "${LOG_FILE}"
+    fi
+
+    echo -e "\nDone setup repos" 2>&1 | tee -a "${LOG_FILE}"
+    exit ;
+}
+
+
+# Check for required arguments
+[ -z "$CNTRLR_USER" ] && echo "ERROR: -U <controller user name> is required." && exit 1;
+[ -z "$CNTRLR_PASSWD" ] && echo "ERROR: -P <controller password> is required." && exit 1;
+[ -z "$STORAGE_TYPE" ] && echo "ERROR: --storage_type <type> is required." && exit 1;
+[ -z "$SERVER_TYPE" ] && echo "ERROR: --server_type <type> is required." && exit 1;
+if [ "$PRODUCT_VERSION" == "LDR_R2" ] &&  [ -z "$BMC_IP" ]
+then
+    skip_bmc=true
+    echo "Skipping BMC information for LDR_R2..." 2>&1 | tee -a "${LOG_FILE}" ;
+else
+    [ -z "$BMC_IP" ] && echo "ERROR: --i <BMC IPV4> is required." && exit 1;
+    [ -z "$BMC_USER" ] && echo "ERROR: --u <BMC user> is required." && exit 1;
+    [ -z "$BMC_PASSWD" ] && echo "ERROR: --p <BMC password> is required." && exit 1;
+fi
+
+
+# Install required RPMS if not available
+"$install_3rd_party_packages" && {
+
+    echo "INFO: INSTALLING python3..." 2>&1 | tee -a "${LOG_FILE}"
+    yum install -y python3 2>&1 | tee -a "${LOG_FILE}"
+
+    echo "INFO: INSTALLING cortx-py-utils..." 2>&1 | tee -a "${LOG_FILE}"
+    yum install -y cortx-py-utils 2>&1 | tee -a "${LOG_FILE}"
+
+    echo "INFO: INSTALLING consul..." 2>&1 | tee -a "${LOG_FILE}"
+    yum install -y consul 2>&1 | tee -a "${LOG_FILE}"
+
+    echo "INFO: INSTALLING rabbitmq-server..." 2>&1 | tee -a "${LOG_FILE}"
+    yum install -y rabbitmq-server 2>&1 | tee -a "${LOG_FILE}"
+
+}
+
+
+# If local RPMS location is specified, SSPL RPMS will be
+# installed from the speicifed path. Otherwise yum repos.
+install_sspl_rpms(){
+    if [ -n "$RPMS_PATH" ]; then
+        echo -e "INFO: Installing SSPL RPMS from local path - ${RPMS_PATH}"
+        sudo yum localinstall -y $RPMS_PATH/cortx-libsspl_sec-2* \
+                            $RPMS_PATH/cortx-libsspl_sec-method_none-2* \
+                            $RPMS_PATH/cortx-sspl-2* \
+                            $RPMS_PATH/cortx-sspl-test-2*
+    else
+        echo "INFO: Installing SSPL RPMS using yum repos..."
+        yum install -y cortx-sspl.noarch
+        yum install -y cortx-sspl-test
+    fi
+    echo "Done installing SSPL RPMS.";
+}
+
+
+setup_consul(){
```

Comment:
Any 3rd party components(like consul) install & setup code is being asked now to put in cortx-utils repo, so other components can re-use. Keep a separate tkt to do this, dont want to delay this task for that.

Reply:
Avoided this method as we don't depend on consul.